modules/system/classes/UpdateManager.php

```diff
@@ -365,21 +365,29 @@ public function uninstall()
     }
 
     /**
-     * Asks the gateway for the lastest build number and stores it.
+     * Asks the composer/gateway for the lastest build number and stores it.
      * @return void
      */
     public function setBuildNumberManually()
     {
-        $postData = [];
+        // Use installed.json to detect latest version
+        $composerInstalled = base_path('vendor/composer/installed.json');
```

Comment:
@Samuell1 Any reason for using this file over using `composer.lock` in the root folder?